test/unit/launcher_test.rb

```diff
@@ -3,12 +3,12 @@
 describe 'Cron launcher' do
   describe 'initialization' do
     before do
-      Sidekiq::Options[:cron_poll_interval] = nil
+      Sidekiq::Cron.configuration.cron_poll_interval = 30
```

Comment:
Maybe we can remove this block? That's already the default value.

Reply:
It fixes some failing tests on CI.

Reply:
But if we are checking the defaults here, I think we should avoid to explicitly set 30, but instead run `Sidekiq::Cron.configure`?

Reply:
Yes, I tried it, locally it works, but it fails here on CI tests.